features/stamp_duty.feature

```diff
@@ -12,51 +12,139 @@ Scenario: Welsh users
   Given I visit the Welsh Stamp Duty page
   Then  I see the Welsh stamp duty calculator
 
-Scenario: House price which is less than £125,000
+Scenario: House price which is less than £40,000 for first home
```

Comment:
These could be better organised using [scenario outlines](https://github.com/cucumber/cucumber/wiki/Scenario-Outlines)

```cucumber
Scenario Outline: stamp duty for first home
  Given I visit the Stamp Duty page
  When I enter my house <price>
  And I click next 
  Then I see the stamp duty I will have to pay is <tax>

  Examples:
    | price  | tax  |
    | 39000  | 0    |
    | 40000  | 0    |
    | 120000 | 0    |
    | 126000 | 20   |
    | 260000 | 3000 |
```